src/mem/slicc/ast/OperatorExprAST.py

```diff
@@ -62,7 +62,15 @@ def generate(self, code, **kwargs):
         # Figure out what the input and output types should be
         if self.op in ("==", "!=", ">=", "<=", ">", "<"):
             output = "bool"
-            if ltype != rtype:
+
+            if (
```

Comment:
I don't think the formatting of this if matches the formatting elsewhere in the file